db/models.py

```diff
@@ -1 +1,50 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return f"{self.name} {self.description}"
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race,
+                             on_delete=models.CASCADE,
+                             related_name="skills")
```

Comment:
related_name should be `skill_set`

Reply:
![image](https://github.com/mate-academy/py-game-models/assets/154452850/41820257-19b7-47f6-8bcb-add61fe7dda4)
What does it mean then? :)

Reply:
`skill_set` is required name by test